docs/modules/ROOT/pages/documentation/payara-server/server-configuration/ssl-certificates.adoc

```diff
@@ -66,6 +66,45 @@ In a default installation, the _http-listener-2_ is responsible for handling the
 
 The information can also be entered in the Web Administration console, by navigating to the page Configurations -> server-config -> HTTP Service -> HTTP Listeners > http-listener-2.
 
+[[loading-certificates-from-multiple-keystores]]
+== Loading certificates from multiple keystores
+
+IMPORTANT: All additional keystores must have the same password as your domain master password.
+
+To add additional keystores to the Payara Server you will need to have your keystore and truststore files in a local directory. You can then configure this either in the admin console or using asadmin commands. If you want to load more than one additional keystore use ':' as a delimiter between file paths.
+
+NOTE: When specifying the key or trust store for a specific listener, this is deemed as an absolute and any additional key or trust stores are not considered.
+
+The new JVM properties used to add additional keystores are:
+----
+-Dfish.payara.ssl.additionalKeyStores
+----
+----
+-Dfish.payara.ssl.additionalTrustStores
+----
+=== Using the Admin Console
+To configure the additional keystore locations in the admin console, head to the *Configurations -> <instance configuration> -> JVM Settings* and on the *JVM Options* tab click *Add JVM Option*.
+
+In the new blank JVM property, you can add the new JVM property and the relative paths to your keystores or truststores respectively.
+
+image::ssl/add-additional-keystores-admin-console.png[Configure additional keystores and truststores in Admin Console]
+
+=== Using Asadmin Commands
+JVM options can be configured using the 'create-jvm-options' asadmin command, you can configure your additional keystore and truststore files like in the following examples. If you are loading in multiple additional keystores via asadmin commands, you will need to use '\:' as the delimiter to avoid creating multiple JVM options.
```

Comment:
```suggestion
JVM options can be configured using the 'create-jvm-options' asadmin command, you can configure your additional keystore and truststore files using this command. If you are loading in multiple additional keystores via asadmin commands, you will need to use '\:' as the delimiter to avoid creating multiple JVM options.
```